ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-ref.yaml

```
@@ -16,6 +16,8 @@ ref:
     name: "release:latest"
   - env: CI_JBS_REQPROCESSOR_IMAGE
     name: "release:latest"
+  - env: CI_SPI_OAUTH_IMAGE
+    name: "service-provider-integration-oauth"
```

I'm not sure if this is allowed. IMO the name of the image specified in the step must be compatible with an image name generated for every other job, e.g. "release:latest". but I might be wrong. let's double check that with pj-rehearse

---

@albarbaro see [this failure](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/45433/rehearse-45433-pull-ci-redhat-appstudio-release-service-main-release-service-e2e/1722249545573732352#1:build-log.txt%3A45)
```suggestion
    name: "release:latest"
```

---

you were right, it failed. what do you suggest to put there?